cardano-cli/test/Test/Golden/Diff.hs

```diff
@@ -0,0 +1,25 @@
+module Test.Golden.Diff (diffVsFile) where
+
+import           Cardano.Prelude
+import           Prelude (String)
+import qualified Prelude
+
+import           Data.Algorithm.Diff (PolyDiff (Both), getGroupedDiff)
+import           Data.Algorithm.DiffOutput (ppDiff)
+import           Hedgehog (MonadTest)
+import           Hedgehog.Extras.Test.Base (failMessage)
+
+diffVsFile
```

Comment:
Let's move this to `cardano-cli/test/Test/Utilities.hs`

Reply:
done